pkgs/games/blobwars/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, pkgconfig, gettext, SDL2, SDL2_image, SDL2_mixer, SDL2_net, SDL2_ttf, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "blobwars";
+  version = "2.00";
+  src = fetchurl {
+    url = "mirror://sourceforge/${pname}/${pname}-${version}.tar.gz";
+    sha256 = "c406279f6cdf2aed3c6edb8d8be16efeda0217494acd525f39ee2bd3e77e4a99";
+  };
+
+  nativeBuildInputs = [ pkgconfig gettext ];
```

Comment:
```suggestion
  nativeBuildInputs = [ pkg-config gettext ];
```

Reply:
Thanks, I didn't realize that `pkgconfig` was an older alias. Fixed.